src/main/java/stsjorbsmod/actions/ExhumeAndMakeEtherealAction.java

```diff
@@ -0,0 +1,137 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.AbstractCard.CardType;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.cards.red.Exhume;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import com.megacrit.cardcrawl.powers.CorruptionPower;
+import stsjorbsmod.cards.cull.Seance;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class ExhumeAndMakeEtherealAction extends AbstractGameAction {
+    private AbstractPlayer p;
+    private static final UIStrings uiStrings;
+    public static final String[] TEXT;
+    private String prependDescription;
+
+    private ArrayList<AbstractCard> exhumes = new ArrayList<>();
+
+
+    public ExhumeAndMakeEtherealAction(String prependDescription) {
+        this.p = AbstractDungeon.player;
+        this.prependDescription = prependDescription;
+        this.setValues(this.p, AbstractDungeon.player, this.amount);
+        this.actionType = ActionType.CARD_MANIPULATION;
+        this.duration = Settings.ACTION_DUR_FAST;
+    }
+
+    public void update() {
+        Iterator<AbstractCard> c;
+        AbstractCard card;
+        if (this.duration == Settings.ACTION_DUR_FAST) {
+            if (AbstractDungeon.player.hand.size() == 10) {
```

Comment:
This should use `BaseMod.MAX_HAND_SIZE` rather than 10 (BaseMod patches the original ExhumeAction to account for this)